### PR TITLE
Maintaining column ordering and providing support for renaming columns

### PR DESCRIPTION
**Pull request by arjunrawal4**

### CODE REVIEW

File: tests/test_data_frame.py [hunk 1]
```diff
@@ -535,6 +535,16 @@ def test_complex_write_query(self):
         assertDataFrameEqualsPandas(agg, base_agg)
         assertDataFrameEqualsPandas(ordered, base_ordered)
 
+    def test_rename_columns(self):
```
Comment (rohankumar42) [inline tests/test_data_frame.py:538]: Could you add another test that uses the original and modified dataframes in downstream computations, ensuring the semantics match those of Pandas?

File: pandasql/core.py [hunk 2]
```diff
@@ -398,6 +398,26 @@ def __setitem__(self, col, value):
         # Update self to new object
         self.__dict__.update(new.__dict__)
 
+    def rename(self, columns):
```
Comment (rohankumar42) [inline pandasql/core.py:401]: So we don't support in-place renames, correct?

File: pandasql/core.py [hunk 3]
```diff
@@ -480,7 +501,7 @@ def __init__(self, source: DataFrame, dest: DataFrame, col: str, value):
             val = val._operation_as_str()
 
         new_column = '{} AS {}'.format(val, self.col)
-        columns = columns.insert(len(columns), new_column)
+        columns = columns.insert(col_index, new_column)
```
Comment (rohankumar42) [inline pandasql/core.py:504]: A small test for checking column-order after updates would be good.

**Review by rohankumar42 — CHANGES_REQUESTED** [review]
Looks good! A few comments. Also, the Travis CI build is broken. Let's fix it before we merge.

File: pandasql/core.py (hunk 2 above)
Comment (arjunrawal4) [reply pandasql/core.py:401]: yes, in place is not supported

File: pandasql/core.py (hunk 3 above)
Comment (arjunrawal4) [reply pandasql/core.py:504]: added

**Pull request merged**